index.html

```diff
@@ -8,7 +8,7 @@
 title: Bioschemas
 # Community Call details
 call-minutes: https://docs.google.com/document/d/1kd5F97ogdiPNhLTnkei-RVR8TC8Ohpc5QSPX3KsfDrk
-call-date: 2022-05-23T15:00UTC
+call-date: 2022-06-25T15:00UTC
```

Comment:
It should be 27th of June, should not it? 25th June is a Saturday, 27th is a Monday. Please double check the date.